packages/models-library/src/models_library/functions.py

```diff
@@ -274,3 +274,14 @@ class FunctionJobCollectionDB(BaseModel):
 
 class RegisteredFunctionJobCollectionDB(FunctionJobCollectionDB):
     uuid: FunctionJobCollectionID
+
+
+class FunctionJobCollectionsListFilters(BaseModel):
+    """Filters for listing function job collections"""
+
+    has_function_id: Annotated[
+        str | None,
```

Comment:
All inputs should be limited/constraint. Add a min/maximum or eventually formatting (if applies) to all input parameter in the API.

Reply:
Use .e.g https://docs.pydantic.dev/2.3/api/types/#pydantic.types.StringConstraints and it should propragate to the API automatically

Reply:
I basically copy-pasted this from the solver filters. I guess these constraints would apply to all UIDs? Does that already exist system-wide?